### PR TITLE
Feature/boeing viz

### PR DESCRIPTION
**Pull request by aleesteele**
Initial set up for the project should be complete, at least in terms of packages and initial file tree. I didn't do anything with the svgs or js files in this PR, in order to keep them as sub-divided as possible.

A few notes: 
- I had to update babel in order to run anything when I was working from home, hence the changes to `package.json`.
- Content in Sass or JS files don't have final content by any means, but this will definitely be updated.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,8 +33,10 @@
   "author": "CSIS iDeas Lab",
   "license": "MIT",
   "devDependencies": {
+    "@babel/core": "^7.1.2",
```
Comment (aleesteele) [inline package.json]: Note: Had to update babel in order to run this on my local machine.

File: gulp_tasks/sass.js [hunk 2]
```diff
@@ -12,5 +13,10 @@ gulp.task('sass', function () {
         browsers: config.sass.autoprefixer.browsers
       })
     ]))
+    .pipe(
```
Comment (aleesteele) [inline gulp_tasks/sass.js]: I'm not entirely sure how gulp-rename works, but added this here.

File: assets/_sass/custom-viz/boeing/boeing.scss [hunk 3]
```diff
@@ -0,0 +1,66 @@
+/*==============================
+=          Animations          =
+==============================*/
+
+@keyframes wobble {
```
Comment (aleesteele) [inline assets/_sass/custom-viz/boeing/boeing.scss]: These are just test animations, not necessarily to be used in the final piece. I was just testing css animations!

File: assets/_js/custom-viz/boeing/waypoints.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+const Waypoint = require('waypoints/lib/noframework.waypoints.js')
+
+const WaypointsJS = () => {
```
Comment (aleesteele) [inline assets/_js/custom-viz/boeing/waypoints.js]: Currently working on a separate branch w/ the beginnings of this. Trying to keep these steps separate as you said, and PR'ing them as needed. If you think there's a better way of going about this process, please let me know!

File: assets/_sass/custom-viz/boeing/map.scss [hunk 5]
```diff
@@ -0,0 +1,3 @@
+#map {
```
Comment (jnschrag) [inline assets/_sass/custom-viz/boeing/map.scss]: Note that this file should be renamed to `_map.scss`. You do not want to create a standalone `map.css` file.

File: gulp_tasks/sass.js (hunk 2 above)
Comment (jnschrag) [reply gulp_tasks/sass.js]: Yes, that's correct.

**Pull request merged**